src/python/dxpy/dx_extract_utils/InputsValidator.py

```diff
@@ -81,22 +85,26 @@ def interpret_conditions(self):
                 method_to_call = self.conditions_funcs.get(condition)
                 getattr(self, method_to_call)(key)
 
-    def throw_message(self, check):
+    def throw_schema_message(self, check):
         type = self.schema.get(check).get("error_message").get("type")
```

Comment:
You might want to store the message here in a separate variable (like you have below in `throw_exit_error`).

Reply:
you mean just storing or passing it as parameter? I considered the second option but then every method would have some repetitive coding extracting that message